src/test/pipe/test_pipe.rs

```diff
@@ -439,8 +439,16 @@ fn test_read_after_write_close_with_empty_buffer() -> Result<(), String> {
             );
         });
 
-        // read fd should be readable
-        assert!(is_readable(read_fd).unwrap());
+        // read fd should be POLLHUP
+        assert_eq!(
+            test_utils::poll_status(read_fd, 0).unwrap(),
+            // shadow doesn't support POLLHUP
```

Comment:
Should there be an issue for this? (Linked here?)

Reply:
Added.